### PR TITLE
#260: Dropped country from Options, allow to choose any language

### PR DESCRIPTION
**Pull request by bonkey**
#260: Dropped country from Options, allow to choose any language

### CODE REVIEW

**Comment by bonkey** [comment]
👍

File: AtlasSDK/AtlasSDK/Models/Config.swift [hunk 1]
```diff
@@ -12,14 +12,11 @@ public struct Config {
...
+    public let salesChannelLocale: NSLocale
+    public let interfaceLocale: NSLocale
 
     public var countryCode: String {
-        return locale.countryCode
-    }
-
-    public var languageCode: String {
-        return locale.languageCode
+        return interfaceLocale.countryCode
```
Comment (hani-ibrahim) [inline AtlasSDK/AtlasSDK/Models/Config.swift]: `interfaceLocale.countryCode` or `salesChannelLocale.countryCode`?
I think it read from `salesChannelLocale`. as the user can't create an address that ship to the country based on interface language but based on the salesChannel

Comment (bonkey) [inline AtlasSDK/AtlasSDK/Models/Config.swift]: Interface locale has **only** language set up from options. Country is **always** from sales channel

Comment (bonkey) [inline AtlasSDK/AtlasSDK/Models/Config.swift]: See https://github.com/zalando-incubator/atlas-ios/blob/3945519a91dd243653b3ccc4ef5c24a4ac5d58d7/AtlasSDK/AtlasSDKTests/ConfigSpec.swift#L57

**Comment by hani-ibrahim** [comment]
👍

**Pull request merged**